src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectProtocolValidator.cs

```diff
@@ -198,8 +197,15 @@ public TimeSpan NonceLifetime
         /// <summary>
         /// Gets or sets a value indicating if a 'sub' claim is required.
         /// </summary>
-        [DefaultValue(false)]
-        public bool RequireSub { get; set; }
+        [DefaultValue(true)]
+        public bool RequireSub { get; set; } = RequireSubByDefault;
```

Comment:
@brentschmaltz I'm hesitant on this one because it feels like a potential breaking behavioral change, but if you're feeling confident that we're not going to break people in an unexpected way, I trust your judgement. :)